airbyte-integrations/bases/base-singer/base_singer/singer_helpers.py

```diff
@@ -163,48 +163,67 @@ def get_catalogs(logger, shell_command: str, sync_mode_overrides: Dict[str, Sync
     @staticmethod
     def read(logger, shell_command, is_message=(lambda x: True), transform=(lambda x: x)) -> Generator[AirbyteMessage, None, None]:
```

Comment:
please remove transform, I don't its usage it is not transparent enough why it is here